src/blocks/homepage-articles/class-wp-rest-newspack-articles-controller.php

```diff
@@ -99,6 +100,7 @@ public function get_items( $request ) {
 		}
 
 		Newspack_Blocks::remove_excerpt_length_filter();
+		Newspack_Blocks::remove_excerpt_ellipsis_filter();
```

Comment:
Same issue, this doesn't match the method name in `class-newspack-blocks.php`.

```suggestion
		Newspack_Blocks::remove_excerpt_more_filter();
```